framework/src/main/java/org/checkerframework/framework/source/SourceChecker.java

```diff
@@ -1676,6 +1965,77 @@ protected final void setLintOption(String name, boolean val) {
     }
   }
 
+  /**
+   * Returns the set of subchecker classes on which this checker depends. Returns an empty set if
```

Comment:
What does it mean to "depend"?  Is this different than being a subchecker?

Reply:
It's a subchecker of this current checker rather than a subchecker of some other checker.